flink-streaming-java/src/main/java/org/apache/flink/streaming/runtime/tasks/StreamTask.java

```diff
@@ -256,19 +259,27 @@
     private final StreamTaskAsyncExceptionHandler asyncExceptionHandler;
 
     /**
-     * Flag to mark the task "in operation", in which case check needs to be initialized to true, so
-     * that early cancel() before invoke() behaves correctly.
+     * INITIALIZED: task constructor was called or on init state. RUNNING: task is in operation.
```

Comment:
How about moving the description of the state to the corresponding state field? It's similar to `JobStatus`[1] or `ExecutionState`[2].

[1] https://github.com/apache/flink/blob/2e91543836d667a0b367688bb5ce290c3164479c/flink-core/src/main/java/org/apache/flink/api/common/JobStatus.java#L26
[2] https://github.com/apache/flink/blob/2e91543836d667a0b367688bb5ce290c3164479c/flink-runtime/src/main/java/org/apache/flink/runtime/execution/ExecutionState.java#L57